packages/apps/human-app/frontend/src/modules/worker/oracle-registration/hooks/use-registration-form.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase -- ... */
+import { useForm } from 'react-hook-form';
+import { useParams } from 'react-router-dom';
+import { zodResolver } from '@hookform/resolvers/zod';
+import {
+  type RegistrationInExchangeOracleDto,
+  registrationInExchangeOracleDtoSchema,
+} from '@/modules/worker/services/registration-in-exchange-oracles';
+
+export function useRegistrationForm() {
+  const { address: oracleAddress } = useParams<{ address: string }>();
```

Comment:
We don't have some complex logic in this hook and simply wrap `useForm`, so I would suggest moving it to the `RegistrationForm` component in order to easily read it and see what are default values, how fields are named (and if they are consistent with what specified in the component)